[2024-03-02T15:54:37.840+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T15:10:00+00:00 [queued]>
[2024-03-02T15:54:37.865+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T15:10:00+00:00 [queued]>
[2024-03-02T15:54:37.867+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2024-03-02T15:54:37.903+0000] {taskinstance.py:1363} INFO - Starting attempt 11 of 12
[2024-03-02T15:54:37.904+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2024-03-02T15:54:37.938+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): Scrapping> on 2024-03-02 15:10:00+00:00
[2024-03-02T15:54:37.951+0000] {standard_task_runner.py:55} INFO - Started process 247 to run task
[2024-03-02T15:54:37.956+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'Batching_Data', 'Scrapping', 'scheduled__2024-03-02T15:10:00+00:00', '--job-id', '258', '--raw', '--subdir', 'DAGS_FOLDER/Batching_Data.py', '--cfg-path', '/tmp/tmpzv8m5gvr']
[2024-03-02T15:54:37.960+0000] {standard_task_runner.py:83} INFO - Job 258: Subtask Scrapping
[2024-03-02T15:54:38.105+0000] {task_command.py:376} INFO - Running <TaskInstance: Batching_Data.Scrapping scheduled__2024-03-02T15:10:00+00:00 [running]> on host 3757548564a1
[2024-03-02T15:54:38.293+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Bob Sebastian
AIRFLOW_CTX_DAG_ID=Batching_Data
AIRFLOW_CTX_TASK_ID=Scrapping
AIRFLOW_CTX_EXECUTION_DATE=2024-03-02T15:10:00+00:00
AIRFLOW_CTX_TRY_NUMBER=11
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-03-02T15:10:00+00:00
[2024-03-02T15:54:52.764+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/sklearn/base.py:338: UserWarning: Trying to unpickle estimator LabelEncoder from version 1.2.2 when using version 1.0.2. This might lead to breaking code or invalid results. Use at your own risk. For more info please refer to:
https://scikit-learn.org/stable/modules/model_persistence.html#security-maintainability-limitations
  UserWarning,

[2024-03-02T15:54:53.954+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Batching_Data.py", line 104, in batching_data
    predicted_class = prediction['prediction']
KeyError: 'prediction'
[2024-03-02T15:54:53.983+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=Batching_Data, task_id=Scrapping, execution_date=20240302T151000, start_date=20240302T155437, end_date=20240302T155453
[2024-03-02T15:54:54.011+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 258 for task Scrapping ('prediction'; 247)
[2024-03-02T15:54:54.085+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2024-03-02T15:54:54.140+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
